doc/source/contribute/developer.rst

```diff
@@ -234,6 +234,13 @@ To modify the SCSS files, follow these steps:
    .. code-block:: bash
 
       python -m pip install -e '.[doc]'
+      make -C doc serve
```

Comment:
We have a section above with `tox -e doc-serve`,  and We already have a developer installation section that includes installing Tox, and since Tox is a dependency and the tox.ini file is up to date, wouldn't it make sense to use tox -e doc-serve consistently instead of the make commands?

Reply:
I dont mind either.. I do not use tox and I saw the Makefile so I did this change.

You can put here the tox version, and keep the `serve` in the makefile until you delete it.

Reply:
Yes I would foster `tox` usage even to serve the doc.

Reply:
i added all the commands in a tabset, keeping tox as recommended one, could you please review @germa89  and let me know if this works for you

Reply:
![Screenshot 2025-05-15 at 09 57 51](https://github.com/user-attachments/assets/f65fb8f2-810e-4992-8070-4998bdc12ca7)